google/cloud/spanner_dbapi/connection.py

```diff
@@ -155,6 +157,14 @@ def database(self):
         """
         return self._database
 
+    @property
+    def autocommit_dml_mode(self):
+        """AutocommitDmlMode of this connection.
```

Comment:
nit: Add information what each value means, and what you can do with this property.